test/built-ins/String/prototype/normalize/return-abrupt-from-this.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 21.1.3.12
+description: >
+  Returns abrupt from ToString(this)
```

Comment:
Like `form` String coercion tests, maybe we can combine these into a single file?